src/lib/y2storage/fake_device_factory.rb

```diff
@@ -609,13 +609,15 @@ def create_lvm_pv(parent, args)
     def create_btrfs(parent, args)
       log.info("#{__method__}( #{parent}, #{args} )")
       default_subvolume = args["default_subvolume"]
-      if default_subvolume
-        blk_device = Y2Storage::BlkDevice.find_by_name(@devicegraph, parent)
+      if default_subvolume && !default_subvolume.empty?
+        blk_device = BlkDevice.find_by_name(devicegraph, parent)
         filesystem = blk_device.filesystem
         if !filesystem || !filesystem.type.is?(:btrfs)
           raise HierarchyError, "No btrfs on #{parent}"
         end
-        filesystem.create_btrfs_subvolume(@default_subvolume)
+        btrfs = Storage.to_btrfs(filesystem)
```

Comment:
Now adapted the fake_device_factory completely to the new API; no more Storage:: or Storage. used anywhere.